src/util/train.hpp

```diff
@@ -0,0 +1,80 @@
+#pragma once
+#include "gta/enums.hpp"
+#include "services/players/player_service.hpp"
+#include "script_global.hpp"
+#include "pointers.hpp"
+#include "natives.hpp"
+#include "script.hpp"
+
+namespace big::train
+{
+	inline auto get_all_vehicles()
+	{
+		std::vector<Vehicle> result;
+		rage::CReplayInterface* CReplayInterface_var = *g_pointers->m_replay_interface; 
+		for (int i = 0; i <= 300; i++)
```

Comment:
< 300 would be better, <= 300 is actually 301 cars

Reply:
agreed, sorted